src/components/DefinitionEntry.js

```diff
@@ -83,10 +87,10 @@ export default class DefinitionEntry extends React.Component {
         {name}
       </span>
     )
-    const revisionTag = revisionUrl ? (
+    const revisionTag = definition.described.urls.version ? (
```

Comment:
should probably use `get(item, 'described.urls.version')` here to be compatible with previous